dist/privacy.js

```diff
@@ -341,5 +341,72 @@ return JSON.stringify({
     }
 
     return results;
+  })(),
+
+  ccpa_link: (() => {
+    const allowedCCPALinkPhrases = [
+      //https://petsymposium.org/popets/2022/popets-2022-0030.pdf page 612
+      'do not sell my personal information',
+      'do not sell my information',
+      'do not sell my info',
+      'do not sell my personal info',
+      'do not sell or share my personal information',
+      'do not sell or share my information',
+      'do not sell or share my info',
+      'do not sell or share my personal info',
+      //https://cppa.ca.gov/faq.html
+      'your privacy choices',
+      'your california privacy choices'
+    ]
+
+    // https://petsymposium.org/popets/2022/popets-2022-0030.pdf page 627
+    const CCPAExclusionPhrases = [
+      'terms',
+      'user agreement',
+      'service agreement',
+      'conditions of use',
+      'terms of usage',
+      'privacy notice',
+      'privacy policy',
+      'privacy & cookies',
+      'preferences',
+      'terms of sale',
+      'login',
+      'terms and conditions apply',
+      'accessibility',
+      'your data in search',
+      'shield',
+      'promo',
+      'campaign',
+      'deal',
+      'ad choice',
```

Comment:
```suggestion
```

@bstandaert-wustl Isn't `Your ad choices` a valid CCPA link?

Reply:
Seems it's some alternative framework.

Reply:
I believe it's usually for https://en.wikipedia.org/wiki/AdChoices